components/spin/index.tsx

```diff
@@ -118,7 +121,7 @@ export default class Spin extends React.Component<SpinProps, any> {
         <Animate
           {...divProps}
           component="div"
-          className={`${prefixCls}-nested-loading`}
+          className={`${prefixCls}-nested-loading ${wrapperClassName}`}
```

Comment:
That's right. Maybe we can use the conditional ternary operator for avoiding this extra space. Something like: 
```
const classNames = wrapperClassName ? prefixCls + '-nested-loading ' + wrapperClassName : prefixCls + '-nested-loading'
```
Are you thinking in this solution?